.coveragerc

```diff
@@ -896,6 +896,11 @@ omit =
     homeassistant/components/telnet/switch.py
     homeassistant/components/temper/sensor.py
     homeassistant/components/tensorflow/image_processing.py
+    homeassistant/components/terncy/__init__.py
+    homeassistant/components/terncy/config_flow.py
```

Comment:
Please don't exclude the configuration flow. Configuration flows need to be tested and need to have 100% test coverage. Please remote this line and add tests.